templates/vue-demo-store/components/account/AccountAddressCard.vue

```diff
@@ -55,6 +56,7 @@ const setDefaultBillingAddress = async () => {
 const removeAddress = async (addressId: string) => {
   try {
     await deleteCustomerAddress(addressId);
+    loadCustomerAddresses();
```

Comment:
I think that this should be added after `try and catch` because we are catching an error related to deleting the address not loading, same with the success

Reply:
hmmm, good point @mdanilowicz, but I also pretty sure it won't be caught because of now `await` there and it's just like "let it happen" afterwards and it was intended.  So what would you prefer better: another try catch for loadCustomAddresses in that case?

Reply:
alright, I'm putting it in finally according our discussion in slack